PeerPrep/src/frontend/pages/LandingPage.tsx

```diff
@@ -0,0 +1,46 @@
+import { Link } from "react-router";
+import LandingImg from "../assets/Images/landing_page_img.png";
+
+function LandingPage() {
+  return (
+    <div>
+      <div className="grid grid-cols-2 gap-2 h-full md:pt-40 lg:pt-20">
+        {/* Left side text */}
+        <div className="flex justify-center items-center">
+          <div className="text-left space-y-5">
+            <p className="peerprep-logo">PeerPrep</p>
+            <div className="text-4xl lg:text-5xl font-bold">
+              <p> Master Tech</p>
+              <p> Interviews</p>
+              <p>Together</p>
+            </div>
+            <p className="text-xl lg:text-3xl">
+              Connect with peers, practice problems, <br />
+              and ace your next technical interview!
+            </p>
+            <div className="grid grid-cols-2 gap-4 md:px-10 lg:px-20">
```

Comment:
The buttons are not left aligned with the text above. This can be done via removing paddings.
```suggestion
            <div className="grid grid-cols-2 gap-4">
```

Reply:
@Nicholas-Cheng-De-Fei for some reason. on my side the buttons are not aligned left with the text above. I see from ur screenshots u dun have such issue. Have you try zoom out, or display in desktop screen instead of split screen?